assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,195 @@
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, Fragment, useCallback } from '@wordpress/element';
+import { getQueryArg } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './timzezone-select';
+import AccountField from './account-field';
+import PropertyField from './property-field';
+import ProfileField from './profile-field';
+
+// import Data from 'googlesitekit-data';
+// const { dispatch, select } = Data;
+
+const AccountCreate = () => {
+	// Uncomment this section after 1311 is merged.
+	/*
+	const isDoingCreateAccount = useSelect(
+		( select ) => {
+			return select( STORE_NAME ).isDoingCreateAccount();
+		},
+		[]
+	);
+	const accountTicketTermsOfServiceURL = useSelect(
+		( select ) => {
+			return select( STORE_NAME ).getAccountTicketTermsOfServiceURL();
+		},
+		[]
+	);
+
+	*/
+	//const { createAccount } = useDispatch( STORE_NAME );
+
+	/*
+	// Redirect if the accountTicketTermsOfServiceURL is set.
+	if ( accountTicketTermsOfServiceURL ) {
+		location = accountTicketTermsOfServiceURL;
+	}
+	*/
+
+	const isDoingCreateAccount = false;
+	const { createAccount } = () => {};
+	const { siteName, siteURL, timezone: tz } = global.googlesitekit.admin;
+	const errorCode = getQueryArg( location.href, 'error_code' );
+
+	// Handle expected provisioning flow error codes.
+	let errorMessage = false;
+	switch ( errorCode ) {
+		case 'user_cancel':
+			errorMessage = __( 'The Terms of Service were not accepted.', 'google-site-kit' );
+			break;
+
+		case 'max_accounts_reached':
+			errorMessage = __( 'The Google Analytics account limit has been reached.', 'google-site-kit' );
+			break;
+
+		case 'backend_error':
+			errorMessage = __( 'Unknown service error.', 'google-site-kit' );
+			break;
+	}
+	const [ error, setError ] = useState( errorMessage );
+
+	const handleSubmit = useCallback( ( accountName, propertyName, profileName, timezone ) => {
+		trackEvent( 'analytics_setup', 'new_account_setup_clicked' );
+		createAccount( {
+			accountName,
+			propertyName,
+			profileName,
+			timezone,
+		} ).then( ( e ) => {
+			const { error: err } = e.payload;
+			if ( err ) {
+				setError( err.message ? err.message : __( 'Unknown error.', 'google-site-kit' ) );
+			}
+		} );
+	} );
+
+	const [ accountName, setAccountName ] = useState( siteName );
+	const [ propertyName, setPropertyName ] = useState( siteURL );
```

Comment:
Per the ACs (and current PHP behavior) this should only be the host, not the full URL.

Reply:
Good point. Updated in https://github.com/google/site-kit-wp/pull/1389/commits/4060a8e591a4536115641be8a16500408b8aea54.